pkg/operator/operator.go

```diff
@@ -45,38 +45,60 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
-var OperatorClient client.ClientInterface
-var operatorStore store.Store
-var clusterID string
-var deployMtxs = map[string]*sync.Mutex{} // key is app id
+var (
+	operator *Operator
+)
+
+type Operator struct {
+	client       client.ClientInterface
+	store        store.Store
+	clusterToken string
+	clusterID    string
+	deployMtxs   map[string]*sync.Mutex // key is app id
+}
+
+func Init(client client.ClientInterface, store store.Store, clusterToken string) *Operator {
```

Comment:
why even use an `Init` function? i'd suggest just having the caller initialize a new operator object (e.g. `Operator{`) and set the necessary exported fields, other fields can remain unexported.